src/Features/JsonPatch/src/Microsoft.AspNetCore.JsonPatch.csproj

```diff
@@ -8,6 +8,8 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <PackageTags>aspnetcore;json;jsonpatch</PackageTags>
     <Nullable>disable</Nullable>
+    <!-- Temporarily suppress APICompat CP1002 which warns on .NET Framework as it can't find load internal assemblies: SMDiagnostics.dll and System.ServiceModel.Internals.dll -->
+    <NoWarn>$(NoWarn);CP1002</NoWarn>
```

Comment:
@ViktorHofer Looks like we're still seeing the error on this project:

> ##[error].dotnet\sdk\10.0.100-alpha.1.25063.8\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.ApiCompat.ValidatePackage.targets(39,5): error CP1002: (NETCORE_ENGINEERING_TELEMETRY=Build) Could not resolve reference 'System.ServiceModel.Internals.dll' in any of the provided search directories.

Reply:
Looking. This needs an APICompat update as MSBuild currently doesn't respect NoWarn in outerbuilds (in which PackageValidation runs): https://github.com/dotnet/msbuild/issues/10873

Will work on it tomorrow.

Reply:
https://github.com/dotnet/sdk/pull/46061/